types/json_ml/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for json_ml 0.0
+// Project: https://github.com/59naga/json-ml
+// Definitions by: Alan Plum <https://github.com/pluma>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="cheerio" />
+
+export type Node = string | [string, ...any[]];
+export function parse(xml: string, trim?: boolean): Node[];
+export function stringify(
+    object: Node[],
```

Comment:
Can this be a single `Node`?

Reply:
No, as it'd be impossible to distinguish from a list of text nodes.

Reply:
@amcasey It's valid input, it just doesn't mean what you would think it does (i.e. it'd match `ul<li style="color: red"></li>` not `<ul><li style="color: red"></li></ul>`).

Reply:
@pluma, you're right that that's not what I expected but it should be allowed if it's valid input, no?